Tutorials/Members-Registration-And-Logins/index-v10.md

```diff
@@ -0,0 +1,360 @@
+---
+versionFrom: 10.0.0
+---
+
+---
+versionFrom: 8.0.0
+meta.Title: "Members Registration and Logins"
+meta.Description: "In this article you can learn about how to create Member registration and login functionality for the frontend of your application."
+needsV9Update: "true"
+---
+
+# Member registration and login 
+
+Using tools available on a fresh install of Umbraco CMS, you are able to create a frontend-based registration and login functions, and restrict access to specific areas of your site based on this system.
+
+By the end of this tutorial you will know how to implement a basic register/login functionality on your website, hide pages from non-logged in members, and assign newly registered members to specific member groups.
+
+
+## Create the necessary macros and partial files
+
+Navigate to the Settings dashboard in the Backoffice, and under Templating section, expand Partial View Macro Files folder and create a new partial view.
+
+![Partial create](images/v8-01-Partial-Macro-Create.png)
+
+Make sure to create this Macro view from available snippets.
+
+![Partial macro snippets](images/v8-02-Partial-Macro-Snippets.png)
+
+Start with the Register Member snippet.
+
+![Partial macro register](images/v8-03-Partial-Macro-Register.png)
+
+Make sure to fill out the name of the partial view (you can call it "Register", for example).
+Afterwards, using the same approach, create a Macro Partial View using the Login snippet and name it "Login".
+
+With that done, navigate to Partial Views folder above Partial View Macro Files, and create a New partial view from snippet.
+This view should be using the Login Status snippet.
+
+![Partial snippets](images/v8-04-Partial-Snippets.png)
+
+These three partials already contain all the necessary code to make the register, login and logout functionality work.
+
+Since the Register and Login were created as macro partials, you will have corresponding Macros with the same names available under the Macros folder above the Templating section.
+Make sure to expand the Macros view, and in Editor Settings enable Login and Register macros to be used in Richtext Editor and the grid.
```

Comment:
```suggestion
With that done, navigate to the Partial Views folder above Partial View Macro Files, and create a New partial view from the snippet.
This view should be using the Login Status snippet.

![Partial snippets](images/v8-04-Partial-Snippets.png)

These three partials already contain all the necessary code to make the register, login, and logout functionality work.

Since the Register and Login were created as macro partials, you will have corresponding Macros with the same names available under the Macros folder above the Templating section.
Make sure to expand the Macros view, and in Editor Settings enable log in and Register macros to be used in Richtext Editor and the grid.
```